templates.mae

```diff
@@ -6131,6 +6771,58 @@ f_m_ct {
   8 5 6 1
   9 7 8 1
   :::
- }
-}
+ } 
+} 
+
+f_m_ct { 
```

Comment:
Does the order of the templates in this file matter?

Reply:
they get matched in order, so theoretically I guess more frequent templates should go first, but in practice I don't think it would make a difference

Reply:
thanks, @ZontaNicola . I guess what I was worried about is if there are races between templates, what happens?